### PR TITLE
Task30: Introduction to Selenium WebDriver

### PR DESCRIPTION
**Pull request by VarvaraKrestyanova**
credentials.properties is not pushed. let me know, please, if you need for review

### CODE REVIEW

File: UnitTesting-master/pom.xml [hunk 1]
```diff
@@ -66,6 +66,18 @@
...
+            <artifactId>selenium-support</artifactId>
+            <version>3.141.59</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-chrome-driver</artifactId>
+            <version>3.141.59</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (EugenBorisik) [inline UnitTesting-master/pom.xml]: My recommendation is to use one artifactId "selenium-java", it contains all necessary dependencies

**Review by EugenBorisik — CHANGES_REQUESTED** [review]
1. Please add separate class with all 8 types of By (they can be on different pages, the goal is to see all of them, there are no needs to use them in the test)
2. Optional (out of task's scope): I see that you have already used page object pattern. According to this pattern each public method must return any value/current page/new page, but not void.
3. Optional (out of task's scope): The best practice is to set implicit waiter once after WebDriver instance initialization and do not change it in runtime. Set it in @BeforeEach method in your case.

**Comment by VarvaraKrestyanova** [comment]
@EugenBorisik please, review

File: UnitTesting-master/src/test/java/seleniumMail/Helpers/ByLocators.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package seleniumMail.Helpers;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ByLocators {
```
Comment (EugenBorisik) [inline UnitTesting-master/src/test/java/seleniumMail/Helpers/ByLocators.java:6]: + By.name, By.partialLinkText

**Review by EugenBorisik — APPROVED** [review]

**Pull request merged**